ibm_watson/natural_language_understanding_v1.py

```diff
@@ -1947,19 +3495,24 @@ class Features():
 
     def __init__(self,
                  *,
+                 classifications: 'ClassificationsOptions' = None,
```

Comment:
If these parameters are fully positional and not key based then I'm not sure we can put this at the top of the list. Can you confirm if these are used positionally or by key?

Reply:
So python offers both positional and key based parameters. If no keys are given, Python will input them in order so this change would force people to use key based parameters for the Features object

Reply:
Most of the SDKs already have this order implemented. Should I reverse it for python?